addons/website_sale_stock/views/website_sale_stock.xml

```diff
@@ -1,5 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+  <template id="assets_frontend" inherit_id="website.assets_frontend" name="Shop">
+      <xpath expr="." position="inside">
+        <script type="text/javascript" src="/website_sale_stock/static/src/js/website_sale_stock.js"></script>
+      </xpath>
+  </template>
+  <template id="products_out_of_stock_warning" inherit_id="website_sale.product" active="False" customize_show="True" name="Out of Stock Warning">
```

Comment:
yes, to set customize option whether we need  to display (check stock) "out of stock warning" or not